### PR TITLE
Add Qubes Builder v2 integration 

### PR DESCRIPTION
**Pull request by fepitre**
Depends on https://github.com/fepitre/qubes-builderv2/pull/21

### CODE REVIEW

**Comment by fepitre** [comment]
PipelineRetry

**Comment by fepitre** [comment]
PipelineRetry

**Comment by marmarek** [comment]
PipelineRetry

File: .qubesbuilder [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+source:
+  files:
+  - url: https://libvirt.org/sources/libvirt-6.6.0.tar.xz
+    signature: https://libvirt.org/sources/libvirt-6.6.0.tar.xz.asc
+    pubkeys: &id001
+    - redhat-jiri-denemark-key.asc
+    - redhat-veillard-key.asc
+  - url: https://libvirt.org/sources/python/libvirt-python-6.6.0.tar.gz
+    signature: https://libvirt.org/sources/python/libvirt-python-6.6.0.tar.gz.asc
+    pubkeys: *id001
```
Comment (marmarek) [inline .qubesbuilder:15]: Better avoid those yaml tricks here (and possibly ban them in the qubes-builder parser).

**Review by marmarek — CHANGES_REQUESTED** [review]

**Pull request merged**